presto-hive/src/main/java/com/facebook/presto/hive/HiveSessionProperties.java

```diff
@@ -415,6 +416,11 @@ public HiveSessionProperties(HiveClientConfig hiveClientConfig, OrcFileWriterCon
                         false,
                         value -> HiveCompressionCodec.valueOf(((String) value).toUpperCase()),
                         HiveCompressionCodec::name),
+                booleanProperty(
+                        USE_PAGEFILE_FOR_HIVE_UNSUPPORTED_TYPE,
+                        "use PAGEFILE format for hive unsupported type",
```

Comment:
Help text could give info on when and why you would enable this option.

Like `Automatically switch to PAGEFILE format for materialized exchange when encountering unsupported types`